nix/pkgs.nix

```diff
@@ -25,9 +25,20 @@ let
 
   # Override some packages and utilities
   pkgsOverlay = import ./overlay.nix;
+
+  # Fix for lack of Android SDK for M1 Macs.
+  system = let
```

Comment:
Lets call this `systemOverride` for clarity,

Reply:
Ok, will change.